src/main/java/picocli/CommandLine.java

```diff
@@ -10038,7 +10025,6 @@ public MethodParam(Method method, int paramIndex) {
             }
             @Override public Annotation[] getDeclaredAnnotations() { return method.getParameterAnnotations()[paramIndex]; }
             @Override public void setAccessible(boolean flag) throws SecurityException { method.setAccessible(flag); }
-            @SuppressWarnings("deprecation")
```

Comment:
`Method.isAccessible` will give deprecation warnings when building on Java 9+.
I believe we want to suppress this.
Try this:

```
%JAVA_11_HOME%\bin\javac -Xlint:all -d c:\Temp picocli\CommandLine.java
```